plugin/evm/vm.go

```diff
@@ -1120,6 +1120,11 @@ func (vm *VM) initBlockBuilding() error {
 	vm.builder = vm.NewBlockBuilder(vm.toEngine)
 	vm.builder.awaitSubmittedTxs()
 
+	var p2pValidators p2p.ValidatorSet = &validatorSet{}
```

Comment:
This will work but might leave confusing logs in `DEBUG` as we log when we drop messages because it's not a validator. An alternative would be to use `p2p.NoOpHandler`